components/Navigation/Navigation.js

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import HomeSVG from "../../public/images/house-solid.svg";
+import Map from "../../public/images/mapIcon.svg";
+import List from "../../public/images/List.svg";
+import Add from "../../public/images/add.svg";
+import Image from "next/image";
+
+function Navigation() {
+  const { pathname } = useRouter();
+
+  return (
+    <Container>
+      <UnorderedList>
```

Comment:
These links have no accessible name. Please add one.
Also the structure of your html is not correct, you should have list items inside your list and then anchor tags inside these list items. You are using list items as links.

Reply:
Thanks for your review here, i made those changes you mentioned. I hope the way i did it is correct.